binding/dotnet/Porcupine/Utils.cs

```diff
@@ -59,6 +59,10 @@ public static string PvLibraryPath(string libName)
             {
                 return Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location), $"lib/{_env}/x86_64/{libName}.dylib");
             }
+            else if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX) && _isArm)
```

Comment:
I thought you were also going to check if they were running on dotnet 6.0 when loading M1 lib?

Reply:
As far as I can tell it detects the arch of the runtime, not the platform - so we are in the clear :)